bundles/tools.vitruv.applications.util.temporary/src/tools/vitruv/applications/util/temporary/other/CorrespondenceRetriever.java

```diff
@@ -0,0 +1,12 @@
+package tools.vitruv.applications.util.temporary.other;
+
+import org.eclipse.emf.ecore.EObject;
+
+/**
+ * To be used as a callback for passing correspondence access from Reactions
+ * to helper classes.
+ */
+public interface CorrespondenceRetriever {
+	public EObject retrieveCorrespondingElement(EObject sourceElement,
+			Class<? extends EObject> correspondingElementType, String tag);
+}
```

Comment:
Is this util really temporary or just placed here in absence of a better place?

Reply:
Hm, good question. Actually, I placed it there because all our utilities are placed in the `*.util.temporary` project and wait for cleanup.

Reply:
@JanWittler Is this fine for you, or do you propose to have it placed somewhere else already now?

Reply:
Since we have to come up with a concept of how to structure our utility classes anyways, I am fine with leaving it like this for now.